documentation/modules/ROOT/pages/configuration/outbox-event-router.adoc

```diff
@@ -205,6 +205,12 @@ transforms.outbox.type=io.debezium.transforms.outbox.EventRouter
 value.converter=io.debezium.converters.ByteBufferConverter
 ----
 
+[WARNING]
+====
+If the connector is configured to emit heartbeat events, an additional configuration option `value.converter.delegate.converter.type` will be required.
```

Comment:
It's not only about heartbeats, but also TX metadata and schema change history (for those connectors supporting it).

Docs should also mention how the delegate converter is configured (see my comment on testing above).